app/models/measurement.rb

```diff
@@ -10,11 +9,8 @@ class Measurement < ActiveRecord::Base
 
   #input: measurement
   def bmi_calc
-    #first = last, because of ordering in model
-    weight = self.weight
-    height = self.person.length
+    height = person.length
```

Comment:
Allright! Just keep in mind that when you're going to show a list of measurements, including BMI, it will be expensive to do so.